source/fundamentals/linq.txt

```diff
@@ -587,6 +588,155 @@ in the Atlas manual. For more examples about running Atlas Vector Search queries
 {+driver-short+}, see :atlas:`Run Vector Search Queries </atlas-vector-search/vector-search-stage/>`
 in the Atlas manual and select :guilabel:`C#` from the language dropdown.
 
+Bitwise Operators
+~~~~~~~~~~~~~~~~~
+
+This version of the {+driver-short+} supports the following bitwise operators in 
+the aggregation pipeline. You can use multiple bitwise operators within the same 
+stage. All operands must be of type ``int`` or ``long``. ``$bitAnd``, ``$bitOr``,
+and ``$bitXor`` all take two or more operands. ``$bitNot`` can only take one
+operand.
+
+The examples for this section use the following documents in a collection called
+``ingredients``:
+
+.. code-block:: json
+
+   { "_id": 1, "name": "watermelon", "price": 5, "count": 1 },
+   { "_id": 2, "name": "onions", "price": 2, "count": 4 },
+   { "_id": 3, "name": "eggs", "price": 5, "count": 12 },
+   { "_id": 4, "name": "potatoes", "price": 3, "count": 0 },
+   { "_id": 5, "name": "pasta", "price": 4, "count": 100 },
+   { "_id": 6, "name": "cheese", "price": 4 }
+
+The following ``Ingredient`` class models the documents in the ``ingredients``
+collection:
+
+.. code-block:: csharp
+
+   public class Ingredient 
+   {
+      public int Id { get; set; }
+
+      public string Name { get; set; }
+
+      public int Price { get; set; }
+      
+      public int Count { get; set; }
+   }
+
+.. note::
+
+   For all bitwise operators, if the operand list contains a missing or undefined
+   value, the entire expression evaluates to ``null``.
+
+
+$bitAnd
++++++++
+
+The ``$bitAnd`` aggregation stage performs a bitwise AND operation on the given
+arguments. The following example shows how to generate a ``$bitAnd`` stage using
+LINQ:
```

Comment:
S: right now, it's not totally clear that by bitAnd you mean the **&** operator. Perhaps this paragraph could be preceded by a short sentence that says something like

> You can use the ``$bitAnd`` operator by connecting two or more clauses with a ``&`` character.

Applies to all sections

Reply:
added the suggested sentence after the first one since it flowed better that way in my opinion, but feel free to push back